releasenotes/notes/34024.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+- 34024
+releaseNotes:
+- |
+  **Fixed** Unable to add an item to a list using overlays in IstioOperator
```

Comment:
I don't think we should have release notes for a comment in the code, its not using facing

Reply:
@howardjohn would you like me to remove the releasenotes? I was not really sure if we need one or not. The issue is that the comment in the code read by users and they followed the comment to author their manifest file. when they follow the what the comments said, they got error, that is why we have this issue open. I can remove it but this is actually user facing in my view. Please let me know what you think. Thanks.